src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1847,11 +1847,12 @@ limitations under the License.
                 <requireCommand>AddGroupIfIdentifying</requireCommand>
             </include>
             <include cluster="Thermostat" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+	    <include cluster="Time Synchronization" client="true" server="true" clientLocked="true" serverLocked="true"></include>
```

Comment:
Time Sync server and client optional `[Matter]`

Reply:
Done